common/scripted_effects/LIDA_dom_effects.txt

```diff
@@ -479,6 +479,126 @@ calculate_dom_success_effect = {
 			}
 		}
 	}
+    save_scope_value_as = {
```

Comment:
Can you fix the formatting here? The spacing is off

Reply:
That's weird. It's perfectly fine on my IDE, it only appears like that on GitHub.

![image](https://github.com/PerplexedPeach/dynamic_affairs/assets/151125161/5328f0bf-d9c4-4305-9543-906e29b184ba)

Reply:
Oh, don't tell me it's supposed to be whitespaces instead of tabs?

Reply:
Fixed